### DISCUSSION TITLE
Is it possible to connect a child node to two parents?

### DISCUSSION THREAD

**Discussion by CaneRandagio1983** [freeplane/freeplane#1308] [Help] [answered]
I have a need, I need to connect a child node to two parents, to be joking we can say a father and a mother lol

In the specific case, I have to connect 'soluton map' to 'mind map' and 'concept map', you know that the union of these two maps form the 'solution map'.

Is it possible to do this?

For now I've used a workaround, I've used two graphical links with arrows from the supposed parents to 'solution map' to indicate the merging of the two in this particular map.

**Comment by Illegit** [comment, marked as answer]
I'm a frequent user, so maybe it is better to receive an answer from an expert: you can use clones, or 1 parent normally connected and the other with a simulated edge link. Use the command search tool in the Help menu to individuate all of the possibilities about these commands.

**Reply by CaneRandagio1983** [reply]
I solved using simulated edge link, thanks
